### PR TITLE
An activity with all helpline number

### PR DESCRIPTION
**Pull request by akshaydhiran**

### CODE REVIEW

**Comment by tanmayrauth** [comment]
please provide screenshots and all checks are not passing

**Comment by akshaydhiran** [comment]
Helplineinfo

**sir, i Don't know why this all checks error comes**

**Comment by tanmayrauth** [comment]
@ankitshaw @pravinyo can you guys check on this ?

File: master/lib/helpline.dart [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              ),
+            ),
+            SizedBox(
+              height: 10,
+              width: 10,
+            ),
+            myWidget(),
+          ],
+        ),
+      ),
```
Comment (ankitshaw) [inline master/lib/helpline.dart:36]: use Open sans for header and Roboto for content.

**Review by ankitshaw — CHANGES_REQUESTED** [review]
UI needs improvement. Use fonts Open sans for header and roboto for content . Use material design principles.  For example.
image

File: master/lib/helpline.dart (hunk 1 above)
Comment (ankitshaw) [reply master/lib/helpline.dart:36]: card layout is needed keeping in mind planned features for future

**Comment by tanmayrauth** [comment]
@akshaysingh5 any update over changes suggested by @ankitshaw ?

**Comment by ankitshaw** [comment]
closing this due to inactivity

**Pull request closed without merging**